src/Athens-Examples/AthensCairoSurfaceExamples.class.st

```diff
@@ -47,7 +47,7 @@ AthensCairoSurfaceExamples class >> exampleDrawTextPath [
 		f unlock.
 	].
 	
-	Display getCanvas drawImage: surf asForm at: 0@0
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?